Jellyfin.Api/Controllers/VideosController.cs

```diff
@@ -253,225 +251,71 @@ public async Task<ActionResult> MergeVersions([FromQuery, Required, ModelBinder(
     /// <response code="200">Video stream returned.</response>
     /// <returns>A <see cref="FileResult"/> containing the audio file.</returns>
     [HttpGet("{itemId}/stream")]
     [HttpHead("{itemId}/stream", Name = "HeadVideoStream")]
     [ProducesResponseType(StatusCodes.Status200OK)]
     [ProducesVideoFile]
-    public async Task<ActionResult> GetVideoStream(
-        [FromRoute, Required] Guid itemId,
-        [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? container,
-        [FromQuery] bool? @static,
-        [FromQuery] string? @params,
-        [FromQuery] string? tag,
-        [FromQuery, ParameterObsolete] string? deviceProfileId,
-        [FromQuery] string? playSessionId,
-        [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? segmentContainer,
-        [FromQuery] int? segmentLength,
-        [FromQuery] int? minSegments,
-        [FromQuery] string? mediaSourceId,
-        [FromQuery] string? deviceId,
-        [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? audioCodec,
-        [FromQuery] bool? enableAutoStreamCopy,
-        [FromQuery] bool? allowVideoStreamCopy,
-        [FromQuery] bool? allowAudioStreamCopy,
-        [FromQuery] bool? breakOnNonKeyFrames,
-        [FromQuery] int? audioSampleRate,
-        [FromQuery] int? maxAudioBitDepth,
-        [FromQuery] int? audioBitRate,
-        [FromQuery] int? audioChannels,
-        [FromQuery] int? maxAudioChannels,
-        [FromQuery] string? profile,
-        [FromQuery] string? level,
-        [FromQuery] float? framerate,
-        [FromQuery] float? maxFramerate,
-        [FromQuery] bool? copyTimestamps,
-        [FromQuery] long? startTimeTicks,
-        [FromQuery] int? width,
-        [FromQuery] int? height,
-        [FromQuery] int? maxWidth,
-        [FromQuery] int? maxHeight,
-        [FromQuery] int? videoBitRate,
-        [FromQuery] int? subtitleStreamIndex,
-        [FromQuery] SubtitleDeliveryMethod? subtitleMethod,
-        [FromQuery] int? maxRefFrames,
-        [FromQuery] int? maxVideoBitDepth,
-        [FromQuery] bool? requireAvc,
-        [FromQuery] bool? deInterlace,
-        [FromQuery] bool? requireNonAnamorphic,
-        [FromQuery] int? transcodingMaxAudioChannels,
-        [FromQuery] int? cpuCoreLimit,
-        [FromQuery] string? liveStreamId,
-        [FromQuery] bool? enableMpegtsM2TsMode,
-        [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? videoCodec,
-        [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? subtitleCodec,
-        [FromQuery] string? transcodeReasons,
-        [FromQuery] int? audioStreamIndex,
-        [FromQuery] int? videoStreamIndex,
-        [FromQuery] EncodingContext? context,
-        [FromQuery] Dictionary<string, string> streamOptions)
+    public async Task<ActionResult> GetVideoStream([FromRoute][Required] Guid itemId, [FromQuery] VideoStreamRequest request)
```

Comment:
We moved away from using a single object for query parameters. Please revert this change.

Reply:
Reverted.